lib/WasmReader/WasmCustomReader.cpp

```diff
@@ -44,5 +44,11 @@ void WasmCustomReader::AddNode(WasmNode node)
     m_nodes.Add(node);
 }
 
+const uint32 WasmCustomReader::GetCurrentFunctionSize() const
```

Comment:
Maybe we should rename to something like `GetEstimatedBytecodeSize()`, because someone might infer that `GetCurrentFunctionSize()` always returns the wasm size

Reply:
I was thinking the same thing.
I renamed it to `EstimateCurrentFunctionBytecodeSize` and made a better estimate for import thunks.